CHANGES.md

```diff
@@ -0,0 +1,2 @@
+0.1 (13-Dec-2015)
```

Comment:
This date format scares and confuses me. I think YYYY-MM-DD is the typical mirage project format and will be the first we parse for in our glorious changelog-automation future.